src/core/financial.jl

```diff
@@ -62,7 +75,7 @@ function Financial(;
         end
     ```
 """
-struct Financial
+mutable struct Financial
```

Comment:
I made Financial mutable because it was necessary for code that @rathod-b has since commented out in commit 6f15ad3. In Scenario we were checking whether any health emissions cost inputs were missing and the user had set include_health_in_objective to true, because if so we need to throw an error. He moved this into the Financial struct by adding include_health_in_objective as an argument in the Financial constructor, which was going to be my approach as well. So, yes, Financial can go back to being immutable.

Reply:
Done in commit 1eda79b and 6b1dcdf. I also changed the defaults for health emissions cost inputs in the Financial constructor from 0.0 back to nothing. That way if the user provides values of 0.0 they won't be overwritten (commit 225c9f5).